Jenkinsfile

```diff
@@ -45,9 +45,9 @@ pipeline {
 
         stage('React Build') {
             agent {
-            docker {
-                label 'linux'
-                image 'node:6-alpine'
```

Comment:
🤯

Reply:
yea i'm surprised it actually compiled at all with 6 but i guess babel takes care of everything